src/profile_manager/templates/profile_manager/password_change_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "profile_manager/base.html" %}
+{% load crispy_forms_tags %}
+
+{% block heading_inner %}{{ heading }}{% endblock %}
+
+{% block content %}
+<form action="" enctype="multipart/form-data" method="post">{% csrf_token %}
+  {{ form|crispy }}
```

Comment:
Can you checkout the signup and signin templates?  Having the fields span the entire width of a fullscreen browser is a bit much.  I think I put the forms in a bootstrap div or something.  Just match those two other views.

Reply:
resized it to col-sm-7